scripts/prepare_sector_network.py

```diff
@@ -2193,24 +2383,24 @@ def add_gas_and_h2_infrastructure(
     if options["gas_network"] or options["H2_retrofit"]:
         gas_pipes = pd.read_csv(clustered_gas_network_file, index_col=0)
 
-    if options["gas_network"]:
-        add_gas_network(
-            n=n,
-            gas_pipes=gas_pipes,
-            options=options,
-            costs=costs,
-            gas_input_nodes_file=gas_input_nodes_file,
-            logger=logger,
-        )
+        if options["gas_network"]:
+            add_gas_network(
+                n=n,
+                gas_pipes=gas_pipes,
+                options=options,
+                costs=costs,
+                gas_input_nodes_file=gas_input_nodes_file,
```

Comment:
This parameter needs to be changed consistently to `gas_input_nodes` including the docstring descriptions in `add_gas_and_h2_infrastructure` and `add_gas_network`. We should take the description from the upstream `add_storage_and_grids` for this:
```
gas_input_nodes : pd.DataFrame
        DataFrame containing gas input node information (LNG, pipeline, etc.)
```
```suggestion
                gas_input_nodes=gas_input_nodes,
```